### ISSUE TITLE
Text that is hyperlinked in a "highlight" callout fails accessibility checks in dark mode

### ISSUE THREAD

**Issue by phrdang** [berkeley-cdss/berkeley-class-site#66]
As mentioned in title, this in a Markdown file fails web accessibility checks in dark mode:

```
{: .highlight }
If sections already exist, reimporting them will result in [duplicate sections](https://github.com/Cal-CS-61A-Staff/berkeley-cs61a/issues/7788). See [Updating existing sections](#updating-existing-sections).
```
